components/inputs/input-number.js

```diff
@@ -1,10 +1,25 @@
 import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { formatNumber, parseNumber } from '@brightspace-ui/intl/lib/number.js';
 import { getUniqueId } from '../../helpers/uniqueId.js';
 import { ifDefined } from 'lit-html/directives/if-defined.js';
 import { inputLabelStyles } from './input-label-styles.js';
 import { inputStyles } from './input-styles.js';
 import { offscreenStyles } from '../offscreen/offscreen.js';
 
+function checkMinMaxValue(value, min, max) {
```

Comment:
nit: `check` makes me think it returns a `boolean` depending on if its within the range. Something like `clampValue` might be a bit more clear.